docs/source/DeveloperGuide/WebAppDIRAC/DevelopWebApp/index.rst

```diff
@@ -27,119 +27,148 @@ Each application consists of two parts:
 
 The folder structure of the server side web installation is as follows:
 
-* <Module name folder such as DIRAC, LHCbDIRAC, WebAppDIRAC>
-
-   * **WebApp**
-      * __init__.py
-      * **handler**: contains all the server side implementations of the framework and all the applications.
-         * __init__.py
-      * **static**: contains all the static content that can be loaded by the client side such as JavaScript files, images and css files
-         * <Module name folder such as DIRAC, LHCbDIRAC, WebAppDIRAC>: contains the client side implementation of each application
-            * Application 1
-            * Application 2
-      * **template**: contains all the templates used by the files in the handler folder
+::
+
+  <Module name such as DIRAC, LHCbDIRAC, WebAppDIRAC>
+  └── WebApp
```

Comment:
Are all these characters the so-called box-drawing-characters? Isn't there a more straight-forward way of achieving the same result?

Reply:
I've just copied that characters used by `tree`. I think it's clearer than the [current bulleted-list style](https://dirac.readthedocs.io/en/latest/DeveloperGuide/WebAppDIRAC/DevelopWebApp/index.html) but I'm open to other suggestions.

Reply:
It's definitely cleaner but just I don't have these characters on the keyboard so the fastest for editing is a copy-paste. I was just wondering if this is the RST-suggested way of working.

Reply:
I didn't mange to find anything better and the Sphinx documentation doesn't include anything like this to use as a reference. The most common other way I've seen is to not bother with the extra characters and use block formatting+indentation. For example from `pytest`:
```
setup.py
mypkg/
    __init__.py
    app.py
    view.py
tests/
    test_app.py
    test_view.py
    ...
```
I tried this but it was harder to keep the distinction between the inline comments and file/folder names.

Reply:
There is apparently also the "ascii" flag for tree using 
https://meta.stackexchange.com/a/147474